beam/beam_job_service.sh

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+readonly LOCAL_JAR_NAME='beam-runners-flink_2.11-job-server.jar'
+readonly SERVICE_INSTALL_DIR='/usr/lib/beam-job-service'
+readonly SERVICE_WORKING_DIR='/var/lib/beam-job-service'
```

Comment:
Are there any situations where the install and working dir are not the same? We should probably merge these, especially since creating/modifying them separately below without anticipating them being the same is error prone.

Reply:
They are not currently the same.  The install dir is in `/usr` while the working dir is in `/var`.